promise-it-wont-hurt-solutions-master/src/05-always-asynchronous.js

```diff
@@ -0,0 +1,9 @@
+var promise = new Promise(function (fulfill, reject) {
+    setTimeout( () => {
+      fulfill('PROMISE VALUE')
+      console.log('MAIN PROGRAM');
```

Comment:
This console.log should be outside the scope of the Promise to test for asynchronous execution. Even though the code is running asynchronously, we won't be able to see it since it'll have after the Promise is resolved.